src/transformation/visitors/binary-expression/destructuring-assignments.ts

```diff
@@ -46,6 +46,19 @@ export function transformDestructuringAssignment(
     }
 }
 
+export function transformAssignmentPattern(
```

Comment:
Is it possible to merge it with `transformDestructuringAssignment`?

Reply:
Moved it up to be used in `transformDestructuringAssignment`

`transformDestructuringAssignment` -> `transformAssignmentPattern` -> ...